docs/4.2. Behaviour - Querying.md

```diff
@@ -16,6 +24,8 @@ Connection upgrade procedures permitting an existing HTTP query to transition to
 
 ### Query API Behaviour
 
+Persistent connections to the Query API are supported via Websockets which can be set up via the /subscriptions resource. Query APIs may additionally support the HTTP 'Upgrade' header sent by clients to upgrade an HTTP GET request to a Websocket. In cases where this is performed, a corresponding entry in /subscriptions must also be created with matching query parameters.
```

Comment:
We don't appear to be very consistent on how we capitalize WebSocket. I believe the protocol should be camel-case singular, but perhaps it's less clear how to do so for one or more connections unless we write "(a) WebSocket connection(s)" everywhere.

Reply:
I'll try and come back to this one. The incorrect version is a little more prevalent in the docs so it will cause this PR to affect far more files if I change it here.